packages/@lwc/engine-core/src/patches/detect-synthetic-cross-root-aria.ts

```diff
@@ -40,6 +40,36 @@ const querySelectorAll = globalThis[
     KEY__NATIVE_QUERY_SELECTOR_ALL
 ] as typeof document.querySelectorAll;
 
+const alreadyLoggedComponentsAndElements: Set<string> = new Set();
+
+// @ts-ignore
+if (process.env.NODE_ENV !== 'production' && typeof __karma__ !== 'undefined') {
+    // @ts-ignore
+    window.__lwcResetAlreadyLoggedSyntheticCrossRootAriaWarnings = () => {
```

Comment:
**Nitpick:** With this new reporting API, I would expect the engine to start to generate more warnings over time. Resulting in an increasing amount of `__lwcResetAlreadyLogged*` scratted around the code base. 

Vite Logger as a specific API logging called [`warnOnce`](https://github.com/vitejs/vite/blob/060cd66878eb6bee53a55ea3ec0afd724ee9b35c/packages/vite/src/node/logger.ts#L120-L123) that is in charge of logging a message only once. We could borrow this idea and have a single `__lwcResetWarnOnce` that would be in charge of resetting this for this entire LWC engine.

Reply:
Done!